src/Makefile.test.include

```diff
@@ -146,8 +146,6 @@ bitcoin_test_clean : FORCE
 	rm -f $(CLEAN_BITCOIN_TEST) $(test_test_bitcoin_OBJECTS) $(TEST_BINARY)
 
 check-local:
-	@echo "Running test/util/bitcoin-util-test.py..."
```

Comment:
As you mentioned in the OP, the python tests will only be run now when running 'make check' from the root dir.

I was going to suggest adding a target "check-full" or so, to run the python checks from src/, but that would effectively give us 3 levels of checking, confusion, and lots more bikeshedding.

So I think it would be wise to not differentiate what "make check" does based on your pwd. Let's call the other tests from here as well.